src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.nav {
+  margin-right: 50px;
+  margin-left: 50px;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+}
+
+.image {
+  margin: 10px 0 10px 50px;
+}
+
+.text {
+  --height: 60px;
+  height: var(--height);
+  line-height: var(--height);
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  color: rgba(0, 0, 0, 1);
+  text-transform: uppercase;
+}
+
+.text:hover {
+  color: rgba(0, 172, 210, 1);
+}
+
+a {
+  text-decoration: none;
+}
+
+.list {
+  display: inline-block;
```

Comment:
your list make the navigation bar too large, annulate margins/paddings

Reply:
I didn't get your this comment!

Reply:
check the mock design 
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/100297341/200584642-e53bc3d4-1e23-4431-b7d0-324803fc7066.png">